hera_pspec/utils.py

```diff
@@ -1002,3 +1004,72 @@ def get_reds(uvd, bl_error_tol=1.0, pick_data_ants=False, bl_len_range=(0, 1e4),
         reds, lens, angs = _reds, _lens, _angs
 
     return reds, lens, angs
+
+def pspecdata_time_difference(ds, time_diff):
+    """
+    Given a PSpecData object and a time difference,  
```

Comment:
seems like you didn't finish your sentence here

Reply:
Would fix it.